seeding.py

```diff
@@ -114,8 +114,21 @@ def get_db():
     return MongoClient(conf.MONGO_URI, uuidRepresentation="standard")[conf.MONGO_DBNAME]
 
 
+def seed_jobs(db, seed_path):
+    with open(seed_path, "r") as f:
+        jobs = json.load(f)
+
+    for job in jobs:
+        job["id"] = uuid4()
+        job['start_date'] = datetime.now()
+        job['published_date'] = datetime.now()
+        job["due_date"] = datetime.now()
+        db.jobs.insert_one(job)
+
```

Comment:
Probably not that important, but i think it would be a bit faster to create the objects and use `insert_many` or something

Reply:
Absolutely. Did just this to optimize another mongodb app, and was way faster :zap: